diff --git a/android/gradle.properties b/android/gradle.properties
index 94adc3a..37389ef 100644
--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
+org.gradle.configuration-cache=false
\ No newline at end of file
diff --git a/lib/screens/login_screen.dart b/lib/screens/login_screen.dart
index 632b04d..1608891 100644
--- a/lib/screens/login_screen.dart
+++ b/lib/screens/login_screen.dart
@@ -34,7 +34,7 @@ class LoginScreenState extends State<LoginScreen> {
         'password': _passwordController.text,
       });
 
-      if (response.statusCode == 200) {
+      if (response.statusCode == 201) {
         ScaffoldMessenger.of(context).showSnackBar(
           const SnackBar(content: Text('Login successful!')),
         );
diff --git a/lib/screens/register_screen.dart b/lib/screens/register_screen.dart
index dba7c58..6f79b17 100644
--- a/lib/screens/register_screen.dart
+++ b/lib/screens/register_screen.dart
@@ -1,5 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:dio/dio.dart';
+import 'dart:developer';
 
 class RegisterScreen extends StatefulWidget {
   const RegisterScreen({super.key});
@@ -25,6 +26,8 @@ class RegisterScreenState extends State<RegisterScreen> {
       _errorMessage = '';
     });
 
+    log('START REGISTRATION HERE');
+
     // Simple validation
     if (_passwordController.text != _confirmPasswordController.text) {
       setState(() {
@@ -36,7 +39,7 @@ class RegisterScreenState extends State<RegisterScreen> {
 
     Dio dio = Dio();
     dio.options.baseUrl =
-        'https://travelapp-api-x5j5.onrender.com'; // Update this with your actual API URL
+        'http://10.0.2.2:3000'; // Update this with your actual API URL
     dio.options.connectTimeout = Duration(milliseconds: 5000);
     dio.options.receiveTimeout = Duration(milliseconds: 5000);
 
@@ -47,7 +50,7 @@ class RegisterScreenState extends State<RegisterScreen> {
         'password': _passwordController.text,
       });
 
-      if (response.statusCode == 200) {
+      if (response.statusCode == 201) {
         ScaffoldMessenger.of(context).showSnackBar(
           const SnackBar(content: Text('Registration successful!')),
         );
diff --git a/lib/service/api_service.dart b/lib/service/api_service.dart
index 42006cc..59ef360 100644
--- a/lib/service/api_service.dart
+++ b/lib/service/api_service.dart
@@ -6,7 +6,7 @@ class ApiService {
   // Constructor for ApiService to initialize Dio with the necessary configurations
   ApiService() {
     _dio.options.baseUrl =
-        'https://travelapp-api-x5j5.onrender.com'; // Your API's base URL
+        'http://10.0.2.2:3000'; // Your API's base URL
     _dio.options.connectTimeout =
         Duration(milliseconds: 20000); // 20 seconds timeout for connection
     _dio.options.receiveTimeout =
@@ -26,16 +26,20 @@ class ApiService {
 
   // Login user
   Future<Response> loginUser(Map<String, dynamic> data) async {
+   
     try {
       final response = await _dio.post('/auth/login', data: data);
 
-      if (response.statusCode == 200) {
+      if (response.statusCode == 201) {
+        print('Login successful: ${response.data}');
         return response; // Successful login
       } else {
         // Handle error response
         throw Exception('Failed to login. Status Code: ${response.statusCode}');
       }
     } catch (e) {
+       print('Error during login: $e');
+
       if (e is DioException) {
         // Check DioError and provide custom error messages
         if (e.response != null) {
